github/ProjectCard.py

```diff
@@ -0,0 +1,175 @@
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+################################################################################
+
+import json
+
+import github.GithubObject
+
+# NOTE: There is currently no current way to get cards "in triage" for a project.
+# https://platform.github.community/t/moving-github-project-cards-that-are-in-triage/3784
+#
+# See also https://developer.github.com/v4/reference/object/projectcard for the next generation GitHub API,
+# which may point the way to where the API is likely headed and what might come back to v3. E.g. ProjectCard.content member.
+class ProjectCard(github.GithubObject.NonCompletableGithubObject):
+    """
+    This class represents Project Cards. The reference can be found here https://developer.github.com/v3/projects/cards
+    """
+
+    def __repr__(self):
+        return self.get__repr__({"id": self._id.value})
+
+    @property
+    def archived(self):
+        """
+        :type: bool
+        """
+        return self._archived.value
+
+    @property
+    def column_url(self):
+        """
+        :type: string
+        """
+        return self._column_url.value
+
+    @property
+    def content_url(self):
+        """
+        :type: string
+        """
+        return self._content_url.value
+
+    @property
+    def created_at(self):
+        """
+        :type: datetime.datetime
+        """
+        return self._created_at.value
+
+    @property
+    def creator(self):
+        """
+        :type: :class:`github.NamedUser.NamedUser`
+        """
+        return self._creator.value
+
+    @property
+    def id(self):
+        """
+        :type: integer
+        """
+        return self._id.value
+
+    @property
+    def node_id(self):
+        """
+        :type: string
+        """
+        return self._node_id.value
+
+    @property
+    def note(self):
+        """
+        :type: string
+        """
+        return self._note.value
+
+    @property
+    def updated_at(self):
+        """
+        :type: datetime.datetime
+        """
+        return self._updated_at.value
+
+    @property
+    def url(self):
+        """
+        :type: string
+        """
+        return self._url.value
+
+    # Note that the content_url for any card will be an "issue" URL, from
```

Comment:
This is a bit unfortunate, but seems like the nature of github issues and pull requests. Note that in a browser you can use a "<repo>/pull/1" style URL to navigate to an issue, and "<repo>/issues/1" to get to a pull request.